src/main/java/gov/usgs/owi/nldi/controllers/LinkedDataController.java

```diff
@@ -79,18 +81,38 @@ public LinkedDataController(LookupDao inLookupDao, StreamingDao inStreamingDao,
 		return rtn;
 	}
 
-	@ApiOperation(value="getFeatures", hidden=true)
+	@ApiOperation(value="getFeatures", response= Feature.class, responseContainer="List")
 	@GetMapping(value="linked-data/{featureSource}", produces=MediaType.APPLICATION_JSON_VALUE)
-	public Object getFeatures(HttpServletRequest request, HttpServletResponse response, @PathVariable(LookupDao.FEATURE_SOURCE) String featureSource) throws IOException {
+	public void getFeatures(HttpServletRequest request, HttpServletResponse response,
+							@PathVariable(LookupDao.FEATURE_SOURCE) @Pattern(regexp=REGEX_VALID_FEATURE_SOURCES) String featureSource) {
 		BigInteger logId = logService.logRequest(request);
+		List<Map<String, Object>> rtn = new ArrayList<>();
+
 		try {
-			response.sendError(HttpStatus.BAD_REQUEST.value(), "This functionality is not implemented.");
+			Map<String, Object> parameterMap = new HashMap<>();
+
+			parameterMap.put(LookupDao.ROOT_URL, "http://localhost:8080/test-url/linked-data");
+			parameterMap.put(LookupDao.FEATURE_SOURCE, featureSource);
+			rtn = lookupDao.getList(BaseDao.FEATURES, parameterMap);
```

Comment:
This will be a small enough dataset that we can just return the List<Map<>> to spring and let it build the JSON and avoid the need to create a transformer. See the getDataSources method in this class.